apps/report-documents-not-received/translations/src/en/validation.json

```diff
@@ -1 +1,17 @@
-{}
+{
+  "dnr-full-name": {
+    "required": "Enter the main applicant's full name",
+    "minlength": "Applicant's full name must be 3 and 250 characters",
```

Comment:
Missing the word 'between' for min & maxlength

Change "Applicant's full name must be 3 and 250 characters" to

"Applicant's full name must be between 3 and 250 characters",

I spoke to UCD about this.